src/flowchem/devices/knauer/knauer_finder.py

```diff
@@ -45,7 +45,7 @@ async def get_device_type(ip_address: str) -> str:
         return "ConnectionError"
     except asyncio.TimeoutError:
         if ip_address == "192.168.1.2":
```

Comment:
Well, I think you have to use logged to show it in the terminal. You can change the phrase, too. It could be more evident that the ip_address is/is not 192.168.1.2 and what it means.

Reply:
I think that the 192.169.1.2 normally assign as the static IP to FlowIR (at least in my computer), please check and delete it